src/CSVImport.php

```diff
@@ -916,8 +916,11 @@ function ParseDate(string $sDate, int $iDateMode): array
     return $aDate;
 }
 
-function GetAge($Month, $Day, $Year): bool
+function GetAge(int $Month, int $Day, int $Year): int
```

Comment:
Since `$Year` is nullable, we should denote that it is nullable or else the typechecker will complain to us

```suggestion
function GetAge(int $Month, int $Day, ?int $Year): int
```